pkg/install/deploystorage.go

```diff
@@ -228,6 +228,22 @@ func (i *Installer) deployStorageTemplate(ctx context.Context, installConfig *in
 							},
 							NotActions: &[]string{
 								"Microsoft.Network/networkSecurityGroups/join/action",
+								"Microsoft.Compute/disks/read",
```

Comment:
Do the `/read` NotActions need to be included here? [From what I can tell](https://docs.microsoft.com/en-us/azure/role-based-access-control/deny-assignments#deny-assignment-properties), the NotActions in a DenyAssignment serve as exceptions for whatever's encompassed by the Actions. Since `*/read` is not listed in the Actions above, I think these NonActions may be unnecessary.